package.json

```diff
@@ -28,14 +28,14 @@
     "@nanostores/react": "^0.7.1",
     "@netlify/functions": "^2.8.2",
     "@octokit/rest": "^21.0.2",
-    "@performant-software/core-data": "^3.0.3",
-    "@performant-software/geospatial": "^3.0.3",
-    "@performant-software/shared-components": "^3.0.3",
+    "@performant-software/core-data": "^3.0.3-beta.1",
```

Comment:
Let's update the `react-components` packages to the latest stable release, `v3.0.4`.